react/.storybook/main.ts

```diff
@@ -4,6 +4,7 @@ const config: StorybookConfig = {
   stories: [
     "../stories/**/*.mdx",
     "../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)",
+    "../lib/**/*.stories.@(js|jsx|mjs|ts|tsx)"
```

Comment:
supporting stories files within library directories such as datepicker that is implemented here